zerver/openapi/zulip.yaml

```diff
@@ -448,6 +448,206 @@ paths:
+                        example: 3
+                      drafts:
+                        type: object
+                        description: |
+                          An dictionary/object mapping the id of each draft the user currently
+                          has to the data for that draft.
+                        additionalProperties:
+                          $ref: "#/components/schemas/Draft"
+                  - example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "count": 3,
+                        "drafts":
+                          {
+                            "1":
+                              {
+                                "type": "stream",
+                                "to": [3],
+                                "topic": "sync drafts",
+                                "content": "Let's add backend support for syncing drafts.",
+                                "timestamp": 1595479019.43915,
+                              },
+                            "2":
+                              {
+                                "type": "private",
+                                "to": [4],
+                                "topic": "",
+                                "content": "What if we made it possible to sync drafts in Zulip?",
+                                "timestamp": 1595479020.43916,
+                              },
+                            "3":
+                              {
+                                "type": "private",
+                                "to": [4, 10],
+                                "topic": "",
+                                "content": "What if we made it possible to sync drafts in Zulip?",
+                                "timestamp": 1595479021.43916,
+                              },
+                          },
+                      }
+    post:
+      operationId: create_drafts
+      tags: ["drafts"]
+      description: |
+        Create one of more drafts for the current user.
+
+        `POST {{ api_url }}/v1/drafts`
+      parameters:
+        - name: drafts
+          in: query
+          description: |
+            A json-encoded list of all of the drafts that should be created for the current user.
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/Draft"
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccess"
+                  - properties:
+                      ids:
+                        type: array
+                        description: |
+                          An array of the IDS for the drafts that were just created in the same
+                          order as they were requested.
```

Comment:
Probably we should document here that if any draft has a validation error, the entire request will fail.

Reply:
Good idea.